### PR TITLE
Release v2.12.6+rev1

### PR DESCRIPTION
**Pull request by floion**

### CODE REVIEW

**Review by spanceac — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Change log
 -----------
 
+# v2.12.6+rev1
```
Comment (spanceac) [inline CHANGELOG.md]: Missing a bracket here

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+## (2018-05-02
+
```
Comment (spanceac) [inline CHANGELOG.md]: Missing a bracket here

Comment (spanceac) [inline CHANGELOG.md]: Missing a bracket here

**Review by spanceac — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (floion) [reply CHANGELOG.md]: Thanks. Fixed

**Comment by floion** [comment]
Superseded by https://github.com/resin-os/resin-iot2000/pull/11

**Pull request closed without merging**